src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -123,12 +123,11 @@ export class ProsemirrorAdapter {
     }
 
     private setupContentConverter() {
-        /* eslint-disable multiline-ternary */
-        this.contentConverter =
-            this.contentType === 'markdown'
-                ? new markdownConverter()
-                : new HTMLConverter();
-        /* eslint-enable multiline-ternary */
+        if (this.contentType === 'markdown') {
+            this.contentConverter = new markdownConverter();
+        } else {
+            this.contentConverter = new HTMLConverter();
+        }
```

Comment:
I updated this commit so that there is no change in functionality, and instead changed the `else` into an `else if` in the commit where you added the error logging 👍